frontend/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,40 @@
+import { IconColor } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import { Icon } from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  children: React.ReactNode;
+  isDisplayed: boolean;
+  title: string;
+  onClose?: () => void;
+};
+
+const Modal: React.FC<Properties> = ({
+  children,
+  isDisplayed,
+  title,
+  onClose,
+}) => {
+  return (
```

Comment:
maybe let's add the react portal to the modal?

Reply:
Thanks for advice, done)

Reply:
@v0ldemar01 After reviewing this comment I changed it with using `createPortal`, but on the meeting @Liza-Veis said that I can just use `<dialog>` component without it

Reply:
yes, as we use dialog, we don't need a portal

Reply:
Contact with guys with Modal, we need to have only one solution to prevent large merge conflicts